### ISSUE TITLE
App not working with LineageOS

### ISSUE THREAD

**Issue by sephii** [veloce/lichobile#1025]
**I'm submitting a** bug report

# Bug report

**Current behavior:**

When running the app on LineageOS (lineage_oneplus3-userdebug 9 PQ3A), it looks like it's loading but it never ends:

![Screenshot_20190826-065135_lichess](https://user-images.githubusercontent.com/754333/63666897-086e0b80-c7d2-11e9-8e8c-d08c3a2a579d.png)

**Expected behavior:**

The spinners disappear and the content is loaded.

**Steps to reproduce:**

Install the app on LineageOS 9 and run it.

**Environment information**

- phone hardware: OnePlus 3T
- phone operating system version: lineage_oneplus3-userdebug 9 PQ3A (nightly 20190819  from https://download.lineageos.org/oneplus3)
- version of lichess mobile: 6.1.4

**Other information:**

Running an `adb logcat` reveals that requests are blocked because of a CORS policy:

```
08-26 07:07:47.729  1355  1355 D SystemWebChromeClient: file:///android_asset/www/index.html: Line 0 : Access to fetch at 'https://lichess.org/' from origin 'file://' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
08-26 07:07:47.729  1355  1355 I chromium: [INFO:CONSOLE(0)] "Access to fetch at 'https://lichess.org/' from origin 'file://' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.", source: file:///android_asset/www/index.html (0)
```

**Comment by Zorbeyd** [comment]
Same at Oneplus 6T with LineageOS 16 (Android 9.0) 

Installed Android System Webview and made my default browser to Chrome. Not solved.

**Comment by sephii** [comment]
Closing it because it's probably a duplicate of https://github.com/veloce/lichobile/issues/897.
